tests/EpisodeTest.php

```diff
@@ -4,36 +4,311 @@
+     *
+     * @return void
+     */
+    public function testAdminUserCanSeeAllEpisodes()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard/episodes'
+            );
+        $this->see($episodes[0]['episode_name']);
+        $this->see($episodes[0]['episode_description']);
+
+        $this->assertViewHas('episodes');
+    }
+
+    /**
+     * assert that an admin user can create a new episode.
+     *
+     * @return void
+     */
+    public function testAdminCanCreateEpisodes()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+
+        $this->actingAs($user)
+             ->call(
+                 'GET',
+                 '/dashboard/episode/create'
+             );
+        $this->see('Create Episode');
+        $this->assertViewHas('channels');
     }
 
+    /**
+     * Assert admin can edit an episode.
+     *
+     * @return void
+     */
+    public function testAdminCanEditEpisode()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard/episode/1/edit'
+            );
+            $this->assertViewHasAll(['episode', 'channels']);
+        $this->actingAS($user)
+            ->call(
+                'PUT',
+                '/dashboard/episode/edit',
+                [
+                    'episode' => 'Swanky updated name',
+                    'description' => 'Swanky updated description',
+                    'episode_id' => 1,
+                    'channel_id' => 1
+                ]
+            );
+        $this->call('GET', '/episodes');
+        $this->assertViewHas('episodes');
+    }
+
+    /**
+     * Assert that an admin can activate an episode.
+     *
+     * @return void
+     */
+    public function testAdminCaActivateEpisode()
+    {
+        $this->withoutMiddleware();
+
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        factory('Suyabay\Episode')->create();
+
+        $this->assertEquals(0, Episode::first()->toArray()['status']);
+        $this->actingAs($user)
+            ->call(
+                'PATCH',
+                '/dashboard/episode/activate',
+                [
+                    'episode_id' => 1
+                ]
+            );
+        $this->assertEquals(1, Episode::first()->toArray()['status']);
+    }
+
+    /**
+     * Assert that an admin can delete an episode.
+     *
+     * @return void
+     */
+    public function testAdminCanDeleteEpisode()
```

Comment:
**Code Quality - Nitpick**

Consistency is Key. In some of the tests above, I noticed you used `testAdminUser.....` then later you switched to just `testAdmin.....` like you did here.

Please choose one pattern and follow through by using it throughout the class